bears/general/TextLintBear.py

```diff
@@ -0,0 +1,101 @@
+import json
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
+                     r'error|warning)\s*(?P<message>.+)')
+class TextLintBear:
+    """
+    The pluggable linting tool for text and markdown. It is similar to
+    ESLint, but textlint for natural language.
+    """
+
+    LANGUAGES = {'HTML', 'Markdown', 'reStructuredText'}
+    REQUIREMENTS = {NpmRequirement('textlint')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar', 'Spelling'}
+    SEE_MORE = 'https://github.com/textlint/textlint'
+
+    @staticmethod
+    def generate_config(filename, file):
+        rules = {
+            'no-todo': True,
+            'no-start-duplicated-conjunction': True,
+            'no-empty-section': True,
+            'date-weekday-mismatch': True,
+            'no-nfd': True,
+            'no-surrogate-pair': True,
+            'ginger': True
+        }
+
+        rules['max-number-of-lines'] = {
+            'max': 300
+        }
+
+        rules['max-comma'] = {
+            'max': 4
+        }
+
+        rules['ng-word'] = {
+            'words': ['word1', 'word2']
+        }
+
+        rules['period-in-list-item'] = {
+            'periodMark': '',
+            'periodMarks': ['.', '。', '．'],
+            'ignoreLinkEnd': True,
+            'allowPeriodMarks': [''],
+            'allowEmoji': False,
+            'forceAppendPeriod': False
+        }
+
+        rules['unexpanded-acronym'] = {
+            'min_acronym_len': 3,
+            'max_acronym_len': 5,
+            'ignore_acronyms': ['OSS']
+        }
+
+        rules['rousseau'] = {
+            'showLevels': ['suggestion', 'warning', 'error'],
+            'ignoreTypes': ['sentence:uppercase']
+        }
+
+        rules['alex'] = {
+            'allow': []
+        }
+
+        rules['common-misspellings'] = {
+            'ignore': []
+        }
+
+        rules['write-good'] = {
```

Comment:
isnt this a separate bear?  again, this bear would need to match that bear settings.